src/elfpy/pricing_models/base.py

```diff
@@ -7,16 +7,11 @@
 
 
 class PricingModel(ABC):
-    """
-    Contains functions for calculating AMM variables
+    """Contains functions for calculating AMM variables
```

Comment:
Nit: Was this intentional? I prefer the """ on a separate line, but you know the correct style better than I do.

Reply:
It's not required, but the examples do it that way: https://peps.python.org/pep-0008/#documentation-strings

I like it because it separates the heading from the rest of the doc strings. It's also how docs are written in a few of the python science computation repos I follow.

The most important thing is that we are consistent so that the auto-generated docs will work.

Reply:
Sure, that makes sense. I'm good with changing to this style.